File: Integrantes/LorenzEzequiel/Laboratorio3/JavaScript/TSLeccion07/Persona.js
class Persona{

    static contadorPersona = 0

    constructor(nombre, apellido, edad){
        this._idPersona = ++Persona.contadorPersona
        this._nombre = nombre
        this._apellido = apellido
        this._edad = edad
    }

    get idPersona(){
        return this._idPersona
    }

    get nombre(){
        this._nombre
    }

    set nombre(nombre){
        this._nombre = nombre
    }

    get apellido(){
        this._apellido
    }

    set apellido(apellido){
        this._apellido = apellido
    }

    get edad(){
        this.edad
    }

    set edad(edad){
        this._edad = edad
    }

    toString(){
        return this._idPersona+' '+this._nombre+' '+this._apellido+' '+this._edad
    }
}

File: Integrantes/PardoUriel/Programacion_3/Leccion07/JavaBeans/src/domain/Persona.java

package domain;

import java.io.Serializable;

public class Persona implements Serializable {

    private String nombre;
    private String appelido;
    
    // Constructor vacio (obligatorio para ser un javabeans)
    public Persona(){
        
    }
    
    public Persona(String nombre, String apellido){
        this.nombre = nombre;
        this.appelido = apellido;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getAppelido() {
        return appelido;
    }

    public void setAppelido(String appelido) {
        this.appelido = appelido;
    }

    @Override
    public String toString() {
        return "Persona{" + "nombre=" + nombre + ", appelido=" + appelido + '}';
    }
}